src/FieldArray.tsx

```diff
@@ -12,14 +12,18 @@ import { getIn, isEmptyChildren, isFunction, setIn } from './utils';
 export type FieldArrayRenderProps = ArrayHelpers & {
   form: FormikProps<any>;
   name: string;
+  [key: string]: any;
```

Comment:
This isn't typed strongly enough. Instead it should be something like `FieldArrayRenderProps<ExtraProps> = ArrayHelpers & {form, name} & ExtraProps;`